fenix/app/src/main/java/org/mozilla/fenix/onboarding/JunoOnboardingFragment.kt

```diff
@@ -169,12 +206,20 @@ class JunoOnboardingFragment : Fragment() {
         )
     }
 
-    private fun shouldShowNotificationPage(context: Context) =
+    private fun canShowNotificationPage(context: Context) =
```

Comment:
👌

Reply:
I felt like we should've stuck with the `shouldShowNotificationPage`. In the our code base, we use `shouldShow` more than `canShow`. I don't think this is something we need to fix, but maybe consider this as a later follow up.

Reply:
I would keep canShow since the method checks something that we cannot really have control over (android version). In my opinion, shouldShow would fit more in a scenario where there are no constraints like this.